README.md

```diff
@@ -29,13 +35,16 @@ Current supported filetypes:
                                        default mapping: `<M-S-F6>`
 
 `:VlsiPasteAsDefinition [{modulename}]` Paste module definition.
-                                       default mapping: `<S-F6>`
+                                       default mapping: `<S-F6>`, `<leader>pe`, `<leader>pm`
 
 `:VlsiPasteAsInterface [{modulename}] ` Paste module interface.
-                                       default mapping: `<C-F6>`
+                                       default mapping: `<C-F6>`, `<leader>pc`
 
 `:VlsiPasteAsInstance [{modulename}]  ` Paste module as bound instance.
-                                       default mapping: `<F6>`
+                                       default mapping: `<F6>`,`<leader>pi`
```

Comment:
See above: either align to [autoload / vlsi.vim](https://github.com/ythonnart/vim-vlsi/pull/1/files#diff-3e098e1f3eb1bba00b4c28baf8a1e8fceef16cb3aed28b1408468acced58a68a) or revert to previous mappings

Reply:
will revert to previous mapping 👍